.github/workflows/release.yml

```diff
@@ -32,17 +35,42 @@ jobs:
             ~/.cargo/registry/cache/
           key: ${{ runner.os }}-cargo-build-stable-${{ hashFiles('**/Cargo.toml') }}
 
-      - uses: chainguard-dev/actions/setup-gitsign@main
-
-      # Pulling rust toolchain & install linux dependencies
-      - uses: dtolnay/rust-toolchain@stable
-
-      - name: Install cargo-release
-        uses: taiki-e/install-action@v1
-        with:
-          tool: cargo-smart-release
+      - name: Git setup
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
 
       - name: Release
         env:
           CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
         run: cargo smart-release -ue
+
+      # Leaving bevy comments here for us to discuss. Do we want to start tagging our pre-releases with `dev`?
```

Comment:
@gmpinder what do you think about using dev tag and then when we release, remove the dev tag

Reply:
I'd be down to do that